resources/authgear/templates/en/web/authflowv2/settings_gender_edit.html

```diff
@@ -0,0 +1,76 @@
+{{ template "authflowv2/__settings_page_frame.html" . }}
+
+{{ define "page-content" }}
+
+{{ template "authflowv2/__navbar.html" (dict
+  "Title" (include "v2.page.settings-profile-edit.default.title-gender" nil )
+  "BackTitle" (include "v2.component.navbar.default.item-back-button-label" nil )
+  "BackHref" (call $.MakeURL "/settings/profile")
+  )
+}}
+
+<form id="form" method="post" data-turbo="false" novalidate
+    class="flex flex-col gap-8 mt-8 items-center w-full">
+  <div class="w-full tablet:max-w-86">
+    {{ $.CSRFField }}
+    {{ template "authflowv2/__settings_radio.html"
+      (dict
+        "Label" (include "v2.page.settings-profile-edit.default.button-label-not-provided" nil)
+        "RadioName" "gender-select"
+        "RadioValue" ""
+        "DefaultChecked" ( not $.Gender )
+      )
+    }}
+    {{ template "authflowv2/__settings_radio.html"
+      (dict
+        "Label" (include "v2.page.settings-profile-edit.default.button-label-gender-female" nil)
+        "RadioName" "gender-select"
+        "RadioValue" "female"
+        "DefaultChecked" ( eq $.Gender "female")
+      )
+    }}
+    {{ template "authflowv2/__settings_radio.html"
+      (dict
+        "Label" (include "v2.page.settings-profile-edit.default.button-label-gender-male" nil)
+        "RadioName" "gender-select"
+        "RadioValue" "male"
+        "DefaultChecked" ( eq $.Gender "male")
+      )
+    }}
+
+    {{ $defaultChecked := false}}
+    {{ if (and (not (eq $.Gender "female")) (not (eq $.Gender "male")) $.Gender) }}
+      {{ $defaultChecked = true }}
+    {{ end }}
+    {{ template "authflowv2/__settings_radio.html"
+      (dict
+        "Label" (include "v2.page.settings-profile-edit.default.button-label-custom" nil)
+        "RadioName" "gender-select"
+        "RadioValue" "other"
+        "CustomHTML" (include "custom_input" .)
+        "DefaultChecked" $defaultChecked
+      )
+    }}
+  </div>
+
+  <button
+    id="save-button"
+    class="primary-btn w-full tablet:max-w-86"
+    type="submit"
+    name="x_action"
+    value="save"
+    data-authgear-event="authgear.button.update_profile"
+  >
+    {{ template "v2.page.settings-profile-edit.default.button-label-save" }}
+  </button>
+</form>
+
+{{ end }}
+
+{{ define "custom_input" }}
+<input
+  class="input radio-input col-span-2"
```

Comment:
Here should not has `radio-input col-span-2`

Reply:
The reason for you to add those class is because you know the implementation detail of the radio component and you know that it will set `radio-input` to visible when the radio checked. This indicate the abstract level is wrong. We should never try to peek the implementation detail of other component and then determine what we implement our stuff